cylc/flow/network/client.py

```diff
@@ -142,18 +142,24 @@ def __init__(self, host, port, encode_method, decode_method, secret_method,
 
         # receive response
         if self.poller.poll(timeout):
-            res = await self.socket.recv_string()
+            if msg['command'].startswith('pb_'):
+                res = await self.socket.recv()
+            else:
+                res = await self.socket.recv_string()
```

Comment:
Feel free to change the previous implementation to `recv`/`bytes` if it makes it easier.

Reply:
Will `self.socket.recv()` work in place of the existing `self.socket.recv_string()`? will there be a extra pre/post process step for the "string" type?

Reply:
Standard `encode`/`decode`.

Reply:
Done.